extension/src/service/web-component-service.js

```diff
@@ -192,6 +202,10 @@ async function fetchAsync(
   requestObj,
   headers
 ) {
+  const moduleConfig = config.getModuleConfig()
+  const removeSensitiveData =
+    requestObj.removeSensitiveData ?? moduleConfig.removeSensitiveData
+  delete requestObj.removeSensitiveData
```

Comment:
Any reason to remove it from request object?

Reply:
The requestObj is later sent to adyen. I don't want to send them unnecessary data.